gdk/Gir.toml

```diff
@@ -421,6 +425,11 @@ status = "generate"
     # the actual method getter after auto-renaming.
     rename = "screen_width_mm"
 
+[[object]]
+name = "Gdk.Seat"
+status = "generate"
+final_type = false
```

Comment:
Yeah the heuristics are going to work better in the new code generator so you don't need to manually set this anymore in most cases

Reply:
We'll see once it's out. ;)

Reply:
Yeah it will take a while, but that one is easy at least